### ISSUE TITLE
Dependency version information is missing

### ISSUE THREAD

**Issue by fenggit** [vanniktech/gradle-maven-publish-plugin#802]
After successfully publishing Android to Maven Central, the Maven Central backend prompts that Dependency version information is missing. How should I configure it?

<img width="1423" alt="image" src="https://github.com/vanniktech/gradle-maven-publish-plugin/assets/4269768/0c6a2356-35fb-4be5-80ed-09bda067de77">

**Comment by fenggit** [comment]
I also set：coordinates("io.github.test1233", "MyTest", "1.0.0")

**Comment by jocoand** [comment]
I solve it by explicitly add the versions to all my library dependencies

```
implementation("androidx.compose.ui")
// update to 
implementation("androidx.compose.ui:1.6.8")
````

**Comment by jocoand** [comment]
My work is by add the version explicitly to all my library dependencies

```
implementation("androidx.compose.ui")
// update to 
implementation("androidx.compose.ui:1.6.8")
````

**Comment by jocoand** [comment]
My workaround is by adding the versions explicitly to all my library dependencies

```
implementation("androidx.compose.ui")
// update to 
implementation("androidx.compose.ui:1.6.8")
````

**Comment by ianthetechie** [comment]
I spent an entire day hunting down a similar issue. I had specified all of my dependency versions, but the Jetpack Compose libs were controlled by a gradle `platform` + BOM package.

Turns out this package isn't available on Maven Central. So I had to add the Google repository manually to the `pom` builder. I don't know why there isn't a helper in the DSL for this, but I had to resort to hand coding. Here's what it looks like:

```groovy
pom {
  // ...
  withXml {
    def repo = asNode().appendNode('repositories').appendNode('repository')
    repo.appendNode('name', 'Google')
    repo.appendNode('id', 'google')
    repo.appendNode('url', ' https://maven.google.com/')
  }
}
```
